apps/ledger-live-desktop/src/renderer/families/aptos/AccountBalanceSummaryFooter.tsx

```diff
@@ -81,19 +80,6 @@ const AccountBalanceSummaryFooter = ({ account }: Props) => {
           <Discreet>{spendableBalance}</Discreet>
         </AmountValue>
       </BalanceDetail>
-      <BalanceDetail>
-        <ToolTip content={<Trans i18nKey="aptos.delegation.delegatedInfoTooltip" />}>
```

Comment:
We may need to remove the translation linked to this key :)

Reply:
I've removed the translation unit.